src/components/WorldListPage_WorldsForEnsOwnersFeature/WorldListPage.tsx

```diff
@@ -239,6 +240,8 @@ const WorldListPage: React.FC<Props> = props => {
       isLoading={isLoading}
       isPageFullscreen={true}
     >
+      <h1>Worlds</h1>
```

Comment:
Is this a placeholder for something else?

Reply:
It's actually part of the design to have that title.